tests/components/do_verify_email.test.jsx

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+import {shallow, configure} from 'enzyme';
+import {browserHistory} from 'react-router';
+
+import Adapter from 'enzyme-adapter-react-16';
```

Comment:
Sorry for this last bit from me. I don't think this is needed here since it's already setup here: https://github.com/mattermost/mattermost-webapp/blob/master/tests/setup.js

Reply:
Alrighty, so just to confirm, before getting this branch approved I need to:
1. Remove `import Adapter from 'enzyme-adapter-react-16';` as it's already been setup in a previous merge
2. Clear up my branch

By clearing up my branch you mean that mess of commits/files changed right? I'm not quite sure what happened, they somehow got included as if they were my commits every time I rebased to the newest `master`, a bit frustrating but I'll look for ways to fix it!

Reply:
Haven't been having much luck with finding a solution for clearing up all those commits and modified files...

Wondering if starting a new branch/pull request would provide a 'cleaner' solution to fixing this up.

Reply:
I temporarily changed the base branch then backed again to master, and it looked like the commits are now cleaner.